src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+    Animal.alive.push(this);
+  }
+
+  isDie() {
```

Comment:
```suggestion
  isDie(herbivore) {
```
This method should get an animal